packages/project-access/src/project/getWebapp.ts

```diff
@@ -0,0 +1,26 @@
+import { join, posix } from 'path';
+import { FileName } from '@sap-ux/project-types';
+import { fileExists, readYAML } from '../file';
+
+interface PartialUi5Yaml {
+    resources?: { configuration?: { paths?: { webapp?: string } } };
+}
+
+/**
+ * Get path to webapp.
+ *
+ * @param projectRoot - root path, where package.json or ui5.yaml is
+ * @returns - path to webapp folder
+ */
+export async function getWebappPath(projectRoot: string): Promise<string> {
+    let webappPath = join(projectRoot, 'webapp');
+    const ui5YamlPath = join(projectRoot, FileName.Ui5Yaml);
+    if (await fileExists(ui5YamlPath)) {
+        const yamlDoc = await readYAML<PartialUi5Yaml>(ui5YamlPath);
```

Comment:
please use the `ui5-config` module to access `ui5*.yaml` files. If some functionality is missing, please add it there.

Reply:
My understanding was that `ui5-config` is used to modify yaml files. In this case I just read it. Another thought I had was: it might be that at some point `ui5-config` requires some functionality from this module and in this case we would create circular dependencies. What do you think?

Reply:
Changed this to use `ui5-config`